src/new/templates/rust/no-ui/hypr-echo/pkg/manifest.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "process_name": "hypr-echo",
+    "process_wasm_path": "/hypr-echo.wasm",
+    "on_exit": "Restart",
+    "request_networking": true,
+    "request_capabilities": [
+      "http-server:distro:sys",
+      "http-client:distro:sys",
+      "vfs:distro:sys",
+      "homepage:homepage:sys"
+    ],
+    "grant_capabilities": [
+      "http-server:distro:sys",
+      "http-client:distro:sys",
+      "vfs:distro:sys",
+      "homepage:homepage:sys"
```

Comment:
Please sort these (and above) alphabetically. In general, if there's no reason for a specific order, alphabetize